syntax = "proto2";

package snippets.proto;

option cc_generic_services = true;

message EchoRequest {
  required string message = 1;
}

message EchoResponse {
  required string message = 2;
}

service EchoService {
  rpc Echo(EchoRequest) returns (EchoResponse);
}
